src/operator/regression_output.cc

```diff
@@ -34,24 +34,59 @@ Operator *RegressionOutputProp<type>::CreateOperator(Context ctx) const {
 DMLC_REGISTER_PARAMETER(RegressionOutputParam);
 
 MXNET_REGISTER_OP_PROPERTY(LinearRegressionOutput, RegressionOutputProp<reg_enum::kLinear>)
-.describe("Use linear regression for final output, this is used on final output of a net.")
-.add_argument("data", "NDArray-or-Symbol", "Input data to function.")
-.add_argument("label", "NDArray-or-Symbol", "Input label to function.")
+.describe(R"code(LinearRegressionOutput computes and optimizes for squared loss.
+
+.. note::
+   Use the LinearRegressionOutput as the final output layer of a net.
+
+A scale can be applied to the gradient by adding parameter `grad_scale`, which is often used in
+multi-loss object function in which we can give different weights to each loss.
+
+)code" ADD_FILELINE)
+.add_argument("data", "Symbol", "Input data to the function.")
+.add_argument("label", "Symbol", "Input label to the function.")
 .add_arguments(RegressionOutputParam::__FIELDS__());
 
 MXNET_REGISTER_OP_PROPERTY(MAERegressionOutput, RegressionOutputProp<reg_enum::kMAE>)
-.describe("Use mean absolute error regression for final output, "
-          "this is used on final output of a net.")
-.add_argument("data", "NDArray-or-Symbol", "Input data to function.")
-.add_argument("label", "NDArray-or-Symbol", "Input label to function.")
+.describe(R"code(MAERegressionOutput function computes mean absolute error.
+
+MAE is a risk metric corresponding to the expected value of the absolute error loss or l1-norm loss.
+
+If :math:`\hat{y}_i` is the predicted value of the i-th sample, and :math:`y_i` is the corresponding true value,
+then the mean absolute error (MAE) estimated over :math:`n` samples is defined as
+
+:math:`\text{MAE}(y, \hat{y} ) = \frac{1}{n} \sum_{i=0}^{n-1} \left| y_i - \hat{y}_i \right|`
+
+.. note::
+   Use the MAERegressionOutput as the final output layer of a net.
+
+A scale can be applied to the gradient by adding parameter `grad_scale`, which is often used in
```

Comment:
What is meant here by a "scale"? This should be clear. Otherwise how can the reader understand "a scale can be applied to the gradient"?